### PR TITLE
[image_picker] [example] fix broken link for image_picker_for_web in image_picker example

### PR DESCRIPTION
**Pull request by piedcipher**
fix a broken link in the example of image_picker.

broken-link: https://pub.dev/packages/image_picker#getting-ready-for-the-web-platform

correct-link: https://pub.dev/packages/image_picker_for_web#limitations-on-the-web-platform

fixes: https://github.com/flutter/flutter/issues/111556

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by piedcipher** [comment]
I'm not sure if a version bump is required here hence I didn't do it.

**Review by tarrinneal — APPROVED** [review]
Good catch! tagging @stuartmorgan for second review

File: packages/image_picker/image_picker/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.8.7+2
 
 * Updates minimum Flutter version to 3.3.
+* Fixed a broken link in the example related to `image_picker_for_web`.
```
Comment (stuartmorgan) [inline packages/image_picker/image_picker/CHANGELOG.md:4]: Nit: Fixes

See the style guide for CHANGELOGs listed in the PR checklist.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/image_picker/image_picker/CHANGELOG.md (hunk 1 above)
Comment (piedcipher) [reply packages/image_picker/image_picker/CHANGELOG.md:4]: my bad, I went through it yesterday but I misread it. Sorry

**Pull request merged**